README.md

```diff
@@ -21,7 +21,8 @@ Installation
 ------------
 See the
 installation guide ([linux](./docs/source/starter/installation.rst),
-[mac](./docs/source/starter/mac_installation.rst))
+[mac](./docs/source/starter/mac_installation.rst),
+[windows](./docs/source/starter/windows_installation.rst))
```

Comment:
In order to get the "release note" action to pass, you'll have to modify docs/source/releases/v1_11_5a0.rst release note file (follow the template of other updates in there when adding notes on these changes)